cli.md

```diff
@@ -46,12 +46,13 @@ These options influence how Guardian attempts to retry failed requests:
 
 Determine the entrypoint directories to run Guardian commands.
 
-Usage: guardian entrypoints [options] <directory>
+Usage: guardian entrypoints [options]
```

Comment:
For consistency can we add a prerequisites for this command also

Reply:
done.